integration-tests/mail/pom.xml

```diff
@@ -46,10 +46,6 @@
 <!--            <groupId>org.apache.camel.quarkus</groupId>-->
 <!--            <artifactId>camel-quarkus-integration-test-mail-mailbox</artifactId>-->
 <!--        </dependency>-->
-        <dependency>
-            <groupId>org.jvnet.mock-javamail</groupId>
-            <artifactId>mock-javamail</artifactId>
-        </dependency>
```

Comment:
This one is managed in integration-tests/pom.xml and should be removed from there too.

Reply:
done